Header header
float32 rtt_min         # ms
float32 rtt_avg         # ms
float32 rtt_max         # ms
float32 rtt_mdev        # ms
float32 packet_loss     # 0-1